coops-ui/src/components/AnnualReport/RegisteredOfficeAddress.vue

```diff
@@ -5,631 +5,372 @@
         <div class="meta-container">
           <label>Delivery Address</label>
           <div class="meta-container__inner">
-            <!-- START: Static Details (Delivery Address) -->
+            <delivery-address
```

Comment:
I noticed that Province is now a drop-down list and I cannot set any value that's not in it. This means I cannot enter a non-Canadian address. Is this OK?

Reply:
This is being discussed separately. This PR can proceed as-is.